Lib/sqlite3/__main__.py

```diff
@@ -58,7 +64,8 @@ def runsource(self, source, filename="<input>", symbol="single"):
             case _:
                 if not sqlite3.complete_statement(source):
                     return True
-                execute(self._cur, source)
+                theme = get_theme(force_no_color=not self._use_color)
```

Comment:
What this means is:
- get the current theme, which is still possibly colorless (in case of no TTY or `$NO_COLOR=1` or `$PYTHON_COLORS=0`);
- but if somebody passed `use_color=False` then force there to be no color.

Reply:
Note that this is the entire theme = all sections.